portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +28,49 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomQuote() {
+    // TODO store quotes in seperate JSON file
+    const greetings =
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+}
+
+function randomizeImage() {
+  // The images directory contains 11 images, so generate a random index between
+  // 1 and 11.
+  const size = 11;
+  const imageIndex = Math.floor(Math.random() * size) + 1;
+  const imgUrl = 'images/grace-' + imageIndex + '.jpg';
+
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
+  imgElement.width = '300';
+
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+
+  // Add Caption
+  var moments = [
+    "text1",
+    "text2",
+    "text3", 
+    "text4",
+    "text5",
+    "text6",
+    "text7",
+    "text8",
+    "text9",
+    "text10",
+    "text11",
+  ]
```

Comment:
Use your moments from data.js instead of these local ones.

Reply:
You can access them from `window.moments`

Reply:
window.moments is undefined when I define window.moments in data.js and call that script.js? Should I be putting a javascript header to link data.js to script.js?

Reply:
Yes

Reply:
Not sure how to do it - I tried doing `<script src="data.js"></script> ` (I realize that is for html) and also tried `import {moments} from 'data'; `

Any tips?

Reply:
Yeah, you need `<script src="data.js"></script>` in your `<head>` tag in the html file. No need for the import statement, as data.js will run (and write this array as a globally variable) as soon as it loads. So once the file loads, `window.moments` will exist in script.js as well.